values.yaml

```diff
@@ -69,6 +69,12 @@ backend:
 
   config:
     mountPath: "/etc/rudderstack"
+    # -- rudder-server config overrides
+    # See [config parameters](https://docs.rudderlabs.com/administrators-guide/config-parameters) for more details
```

Comment:
The doc link provided doesn't exist..

Reply:
Yep - as mentioned above: 

We have deprecated this guide as it had outdated information. Should we just point to the [rudder-config.yaml](https://github.com/rudderlabs/rudderstack-helm/blob/master/rudder-config.yaml) file?